airbyte-integrations/connectors/source-marketo/source_marketo/spec.json

```diff
@@ -8,32 +8,38 @@
     "additionalProperties": false,
     "properties": {
       "domain_url": {
+        "title": "Domain URL",
         "type": "string",
         "description": "Your Marketo Base URL. See <a href=\"https://docs.airbyte.io/integrations/sources/marketo\"> the docs </a> for info on how to obtain this.",
         "examples": ["https://000-AAA-000.mktorest.com"],
         "airbyte_secret": true
       },
       "client_id": {
+        "title": "Client ID",
         "type": "string",
         "description": "Your Marketo client_id. See <a href=\"https://docs.airbyte.io/integrations/sources/marketo\"> the docs </a> for info on how to obtain this.",
         "airbyte_secret": true
       },
       "client_secret": {
+        "title": "Client Secret",
         "type": "string",
         "description": "Your Marketo client secret. See <a href=\"https://docs.airbyte.io/integrations/sources/marketo\"> the docs </a> for info on how to obtain this.",
         "airbyte_secret": true
       },
       "start_date": {
+        "title": "Start Date",
         "type": "string",
         "description": "Data generated in Marketo after this date will be replicated. This date must be specified in the format YYYY-MM-DDT00:00:00Z.",
         "examples": ["2020-09-25T00:00:00Z"],
         "pattern": "^[0-9]{4}-[0-9]{2}-[0-9]{2}T[0-9]{2}:[0-9]{2}:[0-9]{2}Z$"
       },
       "window_in_days": {
+        "title": "Window In Days",
         "type": "integer",
         "description": "The amount of days for each data-chunk begining from start_date. (Min=1, as for a Day; Max=30, as for a Month).",
         "examples": [1, 5, 10, 15, 30],
-        "default": 30
+        "default": 30,
+        "maximum": 30
       }
```

Comment:
How will old exist configs work after this limitation? Will it break them?

Reply:
For Marketo maximum date range is 30 day. Now I add max value to spec to remove possibility enter value more than 30 days. 

Old configs will be works with that limitation.

Reply:
got it. thanks!